packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1250,6 +1250,24 @@ describe('build', () => {
       (await fs.readFile(join(output, 'static/index.txt'), 'utf8')).trim()
     ).toEqual('marketing');
   });
+
+  it('should write to flags.json', async () => {
+    const cwd = fixture('with-flags'); // TODO
+    const output = join(cwd, '.vercel', 'output');
+
+    const exitCode = await build(client);
+    expect(exitCode).toEqual(0);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏fix test

It looks like the sourcemaps are broken or something, but I think this is the line that's failing in the tests right now.

Reply:
Failed because Next 14 requires Node >16, and the tests are using Node 16

Reply:
At least that made it work locally <picture data-single-emoji=":sigh:" title=":sigh:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/sigh/d73ba19d0f61fb4e.png" alt=":sigh:" align="absmiddle"></picture>